content/id/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+
+            <p>Pod merupakan unit terkecil dalam platform Kubernetes. Ketika kita membuat sebuat Deployment, Deployment tersebut membuat Pod dengan kontainer-kontainer di dalamnya (bukannya dengan membuat kontainer secara langsung). Tiap Pod terikat langsung dengan Node di mana dia dijadwalkan dan tetap di sana sampai terminasi (berdasarkan <i>restart policy</i>) atau pengapusan. Jika terjadi kegagalan pada sebuah Node, Pod indentik akan dijadwalkan di Node lain dalam klaster.</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Ringkasan:</h3>
+                    <ul>
+                        <li>Pod</li>
+                        <li>Node</li>
+                        <li>Perintah utama kubectl</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                        <p><i>
+                            Pod merupakan sebuah grup yang terdiri dari satu atau lebih kontainer aplikasi (seperti Docker) dan berisi penyimpanan bersama (volume), alamat IP dan informasi tentang bagaimana menjalankan mereka.
+                        </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">Ikhtisar Pod</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_03_pods.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2>Node</h2>
+                <p>Sebuah Pod selalu berjalan dalam sebuah <b>Node</b>. Node merupakan sebuah mesin pekerja di Kubernetes dan dapat dimungkinkan mesin virtual ataupun fisik, tergantung dari klaster. Tiap Node dikelola oleh Master. Satu Node dapat memiliki beberapa pod, dan master Kubernetes otomatis menangani penjadwalan pod seluruh Node-Node dalam klaster. Penjadwalan otomatis oleh Master memperhitungkan tersedianya sumber daya tiap Node.</p>
+
+                <p>Tiap Node Kuberbetes menjalankan setidaknya:</p>
+                <ul>
+                    <li>Kubelet, satu proses yang bertanggung jawab untuk berkomunikasi antara Master Kuberneter dan Node; ini juga mengelola Pod-Pod dan kontainer-kontainer yang berjalan di sebuah mesin.</li>
+                    <li>Satu waktu larian kontainer (<i>container runtime</i>), seperti Docker, bertanggung jawab untuk menarik <i>image</i> kontainer dari registri, membuka kontainer, dan menjalankan aplikasi.</li>
```

Comment:
No need to translate.
Use "register" official word. https://kbbi.kemdikbud.go.id/entri/register
```suggestion
                    <li>Satu <i>container runtime</i>, seperti Docker, bertanggung jawab untuk menarik <i>image</i> Container dari register, membuka Container, dan menjalankan aplikasi.</li>
```